tests/phpstan/phpstan-1.7.1.2.neon

```diff
@@ -0,0 +1,5 @@
+includes:
+	- %currentWorkingDirectory%/tests/phpstan/phpstan.neon
+
+parameters:
+  ignoreErrors:
```

Comment:
Maybe you can get rid of those?

Reply:
For versions before latest, I can't. Because i should modify the code of PrestaShop for this version.

I remove all I can do.

Reply:
Fixed. Sorry, I check on bad local repository.